js/apps/account-ui/test/my-resources.spec.ts

```diff
@@ -5,7 +5,7 @@ test.describe("My resources page", () => {
   test("List my resources", async ({ page }) => {
     await page.goto("/?realm=photoz");
```

Comment:
this is a problem, we need to login with another realm, but this is the way to do it on the local dev server, not the deployed keycloak

Reply:
Speaking of which this will use account v2 as that is still the default we need to switch it first to run there tests